001-project-file-structure/src/app/states/login/login.controller.js

```diff
@@ -0,0 +1,16 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('angularEx1')
+        .controller('LoginController', LoginController);
+
+    /** @ngInject */
+    function LoginController($state) {
+        var vm = this;
+        vm.login = {userName: ""};
+        vm.submit = function () {
```

Comment:
Remember Hoisting the functions declarations
`vm.submit = submit;
 function submit(){ ... }
`